packages/grafana-e2e-selectors/src/selectors/pages.ts

```diff
@@ -280,9 +280,14 @@ export const Pages = {
   BrowseDashboards: {
     table: {
       body: 'data-testid browse-dashboards-table',
-      row: (uid: string) => `data-testid ${uid} row`,
+      row: (name: string) => `data-testid browse dashboards row ${name}`,
       checkbox: (uid: string) => `data-testid ${uid} checkbox`,
     },
+    NewFolderForm: {
+      form: 'data-testid new folder form',
+      nameInput: 'data-testid new-folder-name-input',
```

Comment:
Just asking coz I haven't added selectors before, any particular reason why `form` has spaces between words and `nameInput` and `createButton` have dashes?

Reply:
just me being inconsistent, but it doesn't make a difference. i'll sort it out